locations/spiders/neoness_fr.py

```
@@ -0,0 +1,16 @@
+
+from locations.categories import Categories
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class NeonessFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "neoness_fr"
+    item_attributes = {
+        "brand": "Neoness",
+        "brand_wikidata": "Q86668014",
+        "extras": Categories.GYM.value,
+    }
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)